### PR TITLE
bugzilla: require verified dependent bugs for 4.6

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/openshift-team-developer-productivity-test-platform @jupierce 
Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -245,6 +245,12 @@ bugzilla:
       - 4.5.z
       target_release: 4.4.z
       validate_by_default: true
+    openshift-4.5:
+      dependent_bug_target_releases:
+      - 4.6.0
+      - 4.6.z
+      target_release: 4.5.z
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml:252]: I *think* until the relesae this should be `4.5.0`? But I'm not sure - I've seen @sdodson knowing stuff like this :)

Comment (sdodson) [reply core-services/prow/02_config/_plugins.yaml:252]: Yeah this should be 4.5.0 until we cut final GA build.

File: core-services/prow/02_config/_plugins.yaml [hunk 2]
```diff
@@ -284,6 +290,12 @@ bugzilla:
       - 4.5.z
       target_release: 4.4.z
       validate_by_default: true
+    release-4.5:
+      dependent_bug_target_releases:
+      - 4.6.0
+      - 4.6.z
+      target_release: 4.5.z
```
Comment (sdodson) [inline core-services/prow/02_config/_plugins.yaml:297]:
```suggestion
      target_release: 4.5.0
```

Comment (droslean) [reply core-services/prow/02_config/_plugins.yaml]: thanks! addressed

**Comment by droslean** [comment]
This PR should also changed the `dependent_bug_states` see: https://github.com/openshift/release/pull/9456

**Pull request merged**